### ISSUE TITLE
[UI/UX] Storymaps Slide Setting UX

### ISSUE THREAD

**Issue by balagurova** [UNDP-Data/geohub#3826]
### Description

**Content'tab**
- Rename ‘Content’ to ‘Card’ in the upper tabs to make it a bit more straightforward.
- Rename ‘Slide Content’ to ‘Text’  to make it a bit more straightforward.
- Rename 'Alignment' to 'Card Alignment'
- Move ‘Hide’ option away from the settings to the slide preview by adding another icon with the eye. 
<img width="827" alt="image" src="https://github.com/user-attachments/assets/f6b91013-04fb-4324-b0d4-635cc2ab1a3a">

- Move the delete icon to the right corner to make it harder to click it accidentally. Also, would be good to add a popup with confirmation of the action since there is no 'back' feature.
- Add a feature to hide the card, so that users can display just the map in some cases. It can be named as 'Card Visibility'. 
- 'Image' item: remove options for the card alignment and place the image in the bottom by default to make the design of storymaps consistent.
<img width="775" alt="image" src="https://github.com/user-attachments/assets/dc7a3ccf-ef1a-4b79-8cc5-41432072b99b">

**Map tab**

- Divide 'Geohub map' and 'Base map' styles into two items to make it easy for users. First, let's have 'Base map style' with four options. Then, 'Data layer' with the geohub map ID selection. 
- Rename 'Map location' and 'Map Animation' to 'Location' and 'Animation' to remove duplicates where they are not necessary. 
- Rename 'Map interactive' to 'Map Controls'.
- Remove 'Rotate Animation' function completely. It is very distracting function from the usability perspective. Let's not allow users to do that :)
- 'Spin Globe' can be removed as well, I guess. Since there is no 'globe' view available. 

Please let me know if you want discuss any of these points :)

**Comment by JinIgarashi** [comment]
@balagurova I agree with your suggestions except the following point

> Divide 'Geohub map' and 'Base map' styles into two items to make it easy for users. First, let's have 'Base map style' with four options. Then, 'Data layer' with the geohub map ID selection.

A chapter only show either basemap or geohub map. Users cannot select both `geohub map` and `base map`. so, I think separating both into two items might not work.

**Comment by balagurova** [comment]
Also, I forgot to mention one thing in this ticket. Right now, in the slide thumbnail the user can see icons only if the slide is active. Would be good to show icons on hover as well for all the slides.

**Comment by JinIgarashi** [comment]
I agree we can remove `Spin globe` since it actually does not work properly. but I would like to keep rotate animation, sometimes rotate animation can be a good visualization.

**Comment by JinIgarashi** [comment]
I fixed them except following two points by #3832:

- Add a feature to hide the card, so that users can display just the map in some cases. It can be named as 'Card Visibility'.
  - What do you think we hide the card if users does not put any title/description and image?
- 'Image' item: remove options for the card alignment and place the image in the bottom by default to make the design of storymaps consistent.
  - I am also thinking this because it is difficult to align image - right,left, center...I will remove image alignment and show image always below description.

**Comment by balagurova** [comment]
> What do you think we hide the card if users does not put any title/description and image?
I think we should not assume that the user will understand that. I suggest using a more straightforward approach by including a toggle to switch the visibility of the card on or off

**Comment by balagurova** [comment]
> I agree we can remove `Spin globe` since it actually does not work properly. but I would like to keep rotate animation, sometimes rotate animation can be a good visualization.

The rotate animation can be pretty distracting and might make it hard, if not impossible, for users to focus on the data. Plus, we already have the fly-in effect, and when you combine that with rotation, it can get really overwhelming and take attention away from the content. So, I really think we should remove this feature to make the experience better for users. 

On the other hand, I think the spin globe effect can be quite nice when (or if) the storymap will allow to use globe visualisation. For example, something like this from the [data.undp.org](https://data.undp.org/)

**Comment by balagurova** [comment]
For the Map Style settings issue we discussed, I would add a more straightforward switcher and labels to guide users on what to do.

The 'Empty Map' is selected by default and it shows a note to 'Choose a map style' with four options.
<img width="368" alt="image" src="https://github.com/user-attachments/assets/884a9e93-9e32-4b40-ab61-0773ebf43e4b">

If the user switches to 'Geohub Map', display a note 'Choose Map from Geohub' with a button that opens a modal. 
<img width="367" alt="image" src="https://github.com/user-attachments/assets/53820a14-dbfe-4881-b71c-281a68f3ab4c">

Let me know what you think about this approach.
